notifications_utils/formatters.py

```diff
@@ -619,8 +623,45 @@ def add_language_divs(_content: str) -> str:
 def remove_language_divs(_content: str) -> str:
     """Remove the tags from content. This fn is for use in the email
     preheader, since this is plain text not html"""
-    content = re.compile(FR_OPEN).sub("", _content)
-    content = re.compile(FR_CLOSE).sub("", content)
-    content = re.compile(EN_OPEN).sub("", content)
-    content = re.compile(EN_CLOSE).sub("", content)
+    return remove_tags(_content, FR_OPEN, FR_CLOSE, EN_OPEN, EN_CLOSE)
+
+
+def add_ga_seal(_content: str) -> str:
+    """
+    Custom parser to add Global Affairs seal logo.
+
+    This is a custom temporary change not meant to exist for more than a few
+    weeks. This should either be removed or upgraded into a full-fledged
+    feature.
+
+    TODO: Review, remove/upgrade this functionality.
+    """
+    ga_seal_regex = re.compile(f"{IMG_GLOBAL_AFFAIRS}")  # matches [[ga-seal]]
+    content = ga_seal_regex.sub(
+        r"""<div style="margin: 20px auto 30px auto;">
+          <img
+            src="https://assets.notification.canada.ca/gc-ga-seal.png"
+            alt="Global Affairs Canada / Affaires mondiales Canada"
+            height="339"
+            width="322"
+          />
+        </div>""",
+        _content,
+    )
+
+    return content
+
+
+def remove_ga_seal(_content: str):
+    return remove_tags(_content, IMG_GLOBAL_AFFAIRS)
```

Comment:
Excuse my lack of python, but does the `*tags` below convert `IMG_GLOBAL_AFFAIRS` into an array automatically?

Reply:
Yes, a bit like Java variable arguments with the ellipsis if you are familiar with it. Python also supports double asterisk for passing through a dictionary as well with keyword arguments, [some examples over here](https://www.programiz.com/python-programming/args-and-kwargs).